src/lib/sql/sql_identifier_resolver.cpp

```diff
@@ -48,19 +53,24 @@ std::shared_ptr<AbstractExpression> SQLIdentifierResolver::resolve_identifier_re
   return matching_expressions[0];
 }
 
-const std::optional<SQLIdentifier> SQLIdentifierResolver::get_expression_identifier(
+const std::vector<SQLIdentifier> SQLIdentifierResolver::get_expression_identifiers(
```

Comment:
I know the `const` was there before, but pls remove it. const of return-by-value makes no sense.